modules/pricing/regions.tf

```diff
@@ -4,7 +4,7 @@ data "aws_regions" "all" {
 
 data "aws_region" "one" {
   #  for_each = data.aws_regions.all.names
-  for_each = { for k, v in data.aws_regions.all.names : k => v if !contains(["ap-northeast-3"], k) }
+  for_each = { for k, v in data.aws_regions.all.names : k => v if !contains(["ap-northeast-3", "ap-southeast-3"], k) }
```

Comment:
I don't think this change is really necessary because `ap-southeast-3` is present in `data.aws_regions.all` for all users (no need to signup/activate like with "Asia Pacific (Osaka-Local)"):

```
"ap-southeast-3" = {
    "description" = "Asia Pacific (Jakarta)"
    "endpoint" = "ec2.ap-southeast-3.amazonaws.com"
    "id" = "ap-southeast-3"
    "name" = "ap-southeast-3"
  }
```

Reply:
I've just fixed it.